cdap-common/src/main/java/co/cask/cdap/common/logging/AuditLogEntry.java

```diff
@@ -70,9 +70,6 @@ public AuditLogEntry(String requestLine, @Nullable String userName,
     this.userName = userName;
     this.clientIP = clientIP;
     this.headers = headers.isEmpty() ? null : Collections.unmodifiableMap(new TreeMap<>(headers));
-
-    this.requestBody = new StringBuilder();
```

Comment:
isn't it simpler to just initialize these fields in the constructor? And then we don't need the null check every time we call `appendRequestBody`

Reply:
then we can't distinguish empty body vs don't log body.

Reply:
ok.